portal/package.json

```diff
@@ -8,6 +8,7 @@
     "chart.js": "^4.4.0",
     "chartjs-plugin-annotation": "^3.0.1",
     "classnames": "^2.3.2",
+    "date-fns": "^3.3.0",
```

Comment:
Because I'm using a functionality that converts a date to a presentation of "yesterday", "7 days ago", "one month ago", etc. It's a very easy usage and "date-fns" delivers it.